packages/react-native/ReactCommon/react/renderer/runtimescheduler/SchedulerPriorityUtils.h

```diff
@@ -50,7 +50,10 @@ static inline std::chrono::milliseconds timeoutForSchedulerPriority(
       return std::chrono::seconds(10);
     case SchedulerPriority::IdlePriority:
       return std::chrono::minutes(5);
+    default:
```

Comment:
I wasn’t very clear here.

We build with Clang “-Wswitch-enum”, which will create a warning/error if switch case does not handle a specific enum value.

That means if we add a new SchedulerPriority, we get a build error to tell us where we need to handle the new case. Unless we have a “default”. Having a default satisfies exhaustiveness, but means that adding support for a new enum value is now a runtime error instead of a compile error.

So, we want to avoid default cases, in any switch where we want new enum values to be handled.

But we can return a value, or fatal, after the switch/case, to satisfy MSVC, and let it know that the function will either return a value, or throw/terminate.

Reply:
Yup, that makes sense.  I didn't really think about it, and just did a search for LOG(FATAL) was used in switches elsewhere.  This is the one I "copied":
https://github.com/facebook/react-native/blob/331d99a94154678848628122e8fe3373ee67fb9b/packages/react-native/ReactAndroid/src/main/jni/react/fabric/FabricMountingManager.cpp#L760
I guess that should probably also be updated to avoid the default, but I'll leave that outside the scope of this change.